### DISCUSSION TITLE
Übersetzungstabelle Bundesländer

### DISCUSSION THREAD

**Discussion by cabiweather** [roe-dl/weewx-DWD#6] [Ideas]
Hallo, 
ich habe das DWD-Plugin nun auch auf meine Wetterseite eingebaut. Vielen Dank, lief einwandfrei.

Da die Station in Hessen ist, musste ich mir erst die Werte zusammensuchen und hätte da ein paar Korrekturen und Ergänzungen für die Doku. Das macht es anderen hoffentlich leichter.
Für die Bundesländer-Tabelle folgende Anmerkungen:
[https://github.com/roe-dl/weewx-DWD/wiki/Abk%C3%BCrzungen-der-Bundesl%C3%A4nder-beim-Deutschen-Wetterdienst]
Hessen DWEG
Pfalz/Saarland DWEI
Bayern DWMG (bereits geklärt)
Bayern2?? = DWOG

zu Hessen die Warncell IDs
106432000 Kreis Darmstadt-Dieburg und Stadt Darmstadt
106412000 Stadt Frankfurt am Main
106631000 Kreis Fulda
106414000 Stadt Wiesbaden
106431000 Kreis Bergstraße
106437000 Odenwaldkreis
106433000 Kreis Groß-Gerau
106439000 Rheingau-Taunus-Kreis
106436000 Main-Taunus-Kreis
106533000 Kreis Limburg-Weilburg
106434000 Hochtaunuskreis
106438000 Kreis und Stadt Offenbach
106440000 Wetteraukreis
106435000 Main-Kinzig-Kreis
106535000 Vogelsbergkreis
106531000 Kreis Gießen
106534000 Kreis Marburg-Biedenkopf
106634000 Schwalm-Eder-Kreis
106632000 Kreis Hersfeld-Rotenburg
106635000 Kreis Waldeck-Frankenberg
106636000 Werra-Meißner-Kreis
906633999 Kreis und Stadt Kassel

Quelle: JSON Datei vom 06.02.2022 als ganz Deutschland Warnungen draußen waren. Da kann man sich das zusammensuchen.

**Comment by roe-dl** [comment]
Ich habe mal das Wiki zum Selber-Ergänzen freigegeben. 

Bei den Abkürzungen scheint sich ja einiges geändert zu haben. Die Werte hatte ich damals alle herausgesucht und kontrolliert.
